File: fq.game_service/src/main/java/com/fq/game_service/dao/Jersey.java
package com.fq.game_service.dao;

import jakarta.persistence.*;

@Entity
public class Jersey {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int id;

    @Column(nullable = false, length = 30)
    private String team;

    @Column(nullable = false)
    private int season;

    @Column(nullable = false)
    private String pictureUrl;

/*
    Getters and Setters
*/

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getTeam() {
        return team;
    }

    public void setTeam(String team) {
        this.team = team;
    }

    public int getSeason() {
        return season;
    }

    public void setSeason(int season) {
        this.season = season;
    }

    public String getPictureUrl() {
        return pictureUrl;
    }

    public void setPictureUrl(String pictureUrl) {
        this.pictureUrl = pictureUrl;
    }
}

File: fq.front_pwa/src/app/models/jersey/jersey.ts
export class Jersey {

    constructor(
        id: number,
        team: string,
        season: number,
        pictureUrl: string
    ) {
        this.id = id;
        this.team = team;
        this.season = season;
        this.pictureUrl = pictureUrl;
    }

    id: number;
    team: string;
    season: number;
    pictureUrl: string;
}
